Stripe3DS2/Stripe3DS2/STDSDeviceInformationParameter.m

```diff
@@ -224,7 +224,7 @@ + (instancetype)deviceName
     return [[STDSDeviceInformationParameter alloc] initWithIdentifier:@"C009"
                                                       permissionCheck:nil
                                                            valueCheck:^id _Nullable{
-                                                               return [UIDevice currentDevice].name;
+                                                               return [UIDevice currentDevice].localizedModel;
```

Comment:
@porter-stripe why localized instead of model?

Reply:
Not a super strong reason besides I _think_ it should be closer to what `name` was since I assume it was localized to some degree based on what the user named their device.

I'm open to either model or localizedModel

Reply:
ah yeah, makes sense.